docs/status/template.md.jinja2

```diff
@@ -1,14 +1,5 @@
-{#
-
-This template/markdown file is auto-generated, please do not edit it directly.
-
-Please edit `branches.json` and raise a pull request, this fille will be
-automatically regenerated on merge.
-
-#}
-
-{% macro badge(repo, branch, action) -%}
-[![{{ action }}](https://github.com/{{ repo }}/actions/workflows/{{ action }}.yml/badge.svg?branch={{ branch }})](https://github.com/{{ repo }}/actions/workflows/{{ action }}.yml)
+{% macro badge(repo, branch, workflow) -%}
```

Comment:
Why rename every instance of action to workflow?!?!?!

Reply:
Because they're workflows not actions

Reply:
Yes and no. With this change there's now a confusing mix of terminology within the same file which is much worse.

Reply:
Is it too late to rename `nightly_actions` and `branch_actions`?

Reply:
If you really want to go through all the downstream workflows, but there are much more worthwhile uses of our time.